sirepo/job_driver/docker.py

```diff
@@ -225,6 +230,21 @@ def _init_hosts(cls):
         assert len(cls.__hosts) > 0, \
             '{}: no docker hosts found in directory'.format(cfg.tls_d)
 
+    def _start_idle_timeout(self):
+        async def _kill_if_idle():
+            if not self.ops:
+                pkdlog('self={}', self)
+                self._idle_timer = None
+                await self.kill()
+            else:
+                self._start_idle_timeout()
+
+        if not self._idle_timer:
+            self._idle_timer = tornado.ioloop.IOLoop.current().call_later(
+                cfg.idle_check_mins * 60,
```

Comment:
let's use secs, since everything else in secs (except run time which is in hours)